docs/docs/reference/gen_notebooks/pii.md

```diff
@@ -348,19 +387,49 @@ print(f"Raw text:\n\t{text_to_anonymize}")
 print(f"Anonymized text:\n\t{anonymized_text}")
 ```
 
-# Applying the Methods to Weave Calls
+### Method 4: Use `autopatch_settings` 
+
+You can use `autopatch_settings` to configure PII handling directly during initialization for one or more of the supported LLM integrations. The advantages of this method are:
+
+1. PII handling logic is centralized and scoped at initialization, reducing the need for scattered custom logic.
+2. PII processing workflows can be customized or disabled entirely for specific intergations.
+
+To use `autopatch_settings` to configure PII handling, define `postprocess_inputs` and/or `postprocess_output` in `op_settings` for any one of the supported LLM integrations. 
+
+```python 
+client = weave.init(
+    ...,
+    autopatch_settings={
+        "openai": {
+            "op_settings": {
+                "postprocess_inputs": ...,
```

Comment:
we can be more specific here and show an action `postprocess_inputs` function.  Maybe something like:

```py
def postprocess(inputs: dict) -> dict:
    if "SENSITIVE_KEY" in inputs:
        inputs["SENSITIVE_KEY"] = "REDACTED"
    return inputs
```
```